worlds/pokemon_crystal/__init__.py

```diff
@@ -680,7 +682,7 @@ def create_item(self, name: str) -> PokemonCrystalItem:
         return self.create_item_by_code(self.item_name_to_id[name])
 
     def get_filler_item_name(self) -> str:
-        item = get_random_filler_item(self.random)
+        item = get_random_filler_item(self.random, self)
```

Comment:
```suggestion
        item = get_random_filler_item(self)
```
Same here, can just pass the `World`